canvas_modules/harness/cypress/support/canvas/node-cmds.js

```diff
@@ -212,3 +235,25 @@ Cypress.Commands.add("getNodeDimensions", (nodeLabel) => {
 		return nodeDimensions;
 	});
 });
+
+Cypress.Commands.add("selectAllNodesUsingShiftKey", () => {
+	cy.get("#canvas-div-0").find(".node-image")
+		.then((nodes) => {
+			cy.useShiftKey()
+				.then((shiftKey) => {
+					// Press and hold the shift key
+					cy.get("body")
+						.type(shiftKey, { release: false });
+
+					// Click all the nodes
+					nodes.each((idx, node) => {
+						cy.wrap(node)
+							.click();
+					});
+
+					// Cancel the shift key press
+					cy.get("body")
+						.type(shiftKey, { release: true });
+				});
+		});
+});
```

Comment:
I just realized this not strictly correct. The shift key is supposed to be used to select a range of nodes in a flow from a start to end node and all those in between.  To select all the nodes on a canvas the user should press the Cmnd/Ctrl key and then click each node in turn.  The reason this works is that as we select each node with shift it does not deselect and previously selected nodes -- which, to be honest, I was surprised by when I tested it since I think it should. So this behavior may change in the future.

So I think this function ought to be changed to be using the Cmnd/Ctrl key instead of Shift.